src/bidiMapper/domains/input/ActionDispatcher.ts

```diff
@@ -69,14 +73,25 @@ async function getElementCenter(
   return {x: x as number, y: y as number};
 }
 
+async function isMacOS(context: BrowsingContextImpl) {
+  const {result} = await (
+    await context.getOrCreateSandbox(undefined)
+  ).callFunction(IS_MAC_DECL, {type: 'undefined'}, [], false, 'none', {});
+  assert(result.type !== 'exception');
+  assert(result.result.type === 'boolean');
```

Comment:
What is the side effect of this assert during runtime? Would the mapper tab crash?

Reply:
It throws an unknown error.